Makefile

```diff
@@ -47,18 +47,24 @@ endif
 export GOFLAGS:=$(GOFLAGS)
 export CGO_ENABLED=0
 
+BINDIR:=/usr/bin/
+
+build: out
+	@echo "> Building..."
+	$(GOCMD) build -ldflags "-s -w -X 'github.com/buildpacks/pack.Version=${PACK_VERSION}' -extldflags ${LDFLAGS}" -trimpath -o ./out/$(PACK_BIN) -a ./cmd/pack
```

Comment:
I'm assuming this has to do with the build tools executing `make` by default so it `build` needs to be the first target. Is there no way of tweaking it? If it's important that this be first, we should comment on it.

Reply:
We can tweak this, but it means we have to add a lot of complexity to the `/.github/workflows/delivery/ubuntu/rules` file. To manually run other build tools. Figured this was a work while trade off.

Reply:
That makes sense. Can we add some notes about it on the target?

For example, "# NOTE: this target must be listed first in order for it to be the default target so that tools like `apt-get` can build it without additional parameters".